pages/saved.tsx

```diff
@@ -0,0 +1,68 @@
+import { NextPage, GetStaticProps } from 'next'
+
+import { Service, getServices } from '../cms/services'
+import { CardList } from '../src/Components/CardList'
+import { HeaderComponent } from '../src/Components/Header'
+import { Layout } from '../src/Components/Layout'
+import { ServicePreview } from './index'
+
+interface ListPageProps {
+  services: Array<ServicePreview>
+}
+
+export const SavedPage: NextPage<ListPageProps> = ({ services }) => {
+  return (
+    <>
+      <Layout>
+        <HeaderComponent />
+        <CardList services={services} listType="saved" />
+      </Layout>
+    </>
+  )
+}
+
+export default SavedPage
+
+export const getStaticProps: GetStaticProps = async () => {
+  const allServices = getServices()
+
+  const allServicePreviews: Array<ServicePreview> = allServices.map(
+    (service: Service) => {
+      const servicePreview = {
+        id: service.id,
+        title: service.title,
+        shortDescription: service.shortDescription,
+        costValue: service.costValue,
+        saved: false,
+
+        ...(service.image?.image && {
+          image: {
+            image: service.image.image,
+            imageAlt: service.image?.imageAlt || '',
+          },
+        }),
+
+        ...(service.categories && { categories: service.categories }),
+
+        ...(service.age && {
+          age: service.age,
+        }),
+
+        ...(service.costQualifier && {
+          costQualifier: service.costQualifier,
+        }),
+
+        ...(service.format && {
+          format: service.format,
+        }),
+      }
+      return servicePreview
+    }
+  )
```

Comment:
Isn't this just all copy-pasted over from the `index.tsx` page? Is there any way we can reduce duplication here?

Reply:
It is.. I think I'm happy to wait until we know things won't be different to optimise here.